spine-godot/spine_godot/SpineSkeleton.cpp

```diff
@@ -113,26 +113,34 @@ void SpineSkeleton::set_slots_to_setup_pose() {
 Ref<SpineBone> SpineSkeleton::find_bone(const String &name) {
 	SPINE_CHECK(skeleton, nullptr)
 	if (EMPTY(name)) return nullptr;
-	auto bone = skeleton->findBone(SPINE_STRING(name));
+	auto bone = skeleton->findBone(SPINE_STRING_TMP(name));
```

Comment:
How often per frame do you call this for this change to make a difference performance wise? It would be great to have a test scene that's close to your real-world scenario, where we can measure any improvements. Profilers don't always tell the truth.

Reply:
`SpineSkeleton::find_bone` wasn't called by us manually, but the function below it `SpineSkeleton::find_slot` is called internally in `SpineSlotNode::update_transform` which gets hit very often.     I just though applying the same optimization to  `find_bone` as `find_slot` was just kind defensive performance coding as I could see it could be source of performance issues as well.